assets/js/view/spreadsheet.rb

```diff
@@ -70,7 +72,21 @@ def render_corporation(corporation)
           }
         }
       props = { style: {} }
-      props[:style]['background-color'] = 'rgba(220,220,220,0.4)' unless corporation.floated?
+
+      market_props = { style: {} }
+
+      if corporation.floated?
```

Comment:
corpoaration.floated? && (color = StockMarket::COLOR_MAP[corporation.share_price.color])

Reply:
I don't think this is correct. If you look at the else, it greys out the line for unfloated companies. so if you add another if here, it will grey out cases where there is not a color is well..which is not correct, as certain floated companies do not have colors set yet

Reply:
ok then

unless corporation.floated?
 props[:style][bgc] = 
elsif (color = ...)